highfidelity/hifi
# Flickable web pages

https://highfidelity.fogbugz.com/f/cases/6610/Add-Momentum-to-Scrolling

For preview only:

Test plan:
- Open Tablet
- Goto Market/Info card on Address bar/Create menu lists, properties etc
- try to scroll web page. Make sure its flickable

Known issues:

HTML tables scrolling possible only by clicking on scroller
